HealthcareSystem/PatientWebApp/Controllers/FeedbackController.cs

```diff
@@ -21,11 +21,19 @@ public FeedbackController(IFeedbackService feedbackService)
             _feedbackService = feedbackService;
         }
 
-        [HttpPost("feedbacks")]
-        public void AddFeedback(FeedbackDTO feedbackDTO)
+       [HttpPost]
+        public ActionResult AddFeedback(FeedbackDTO feedbackDTO)
         {
-            Feedback feedback = FeedbackAdapter.FeedbackDTOToFeedback(feedbackDTO);
-            _feedbackService.AddFeedback(feedback);
+            try
+            {
+                Feedback feedback = FeedbackAdapter.FeedbackDTOToFeedback(feedbackDTO);
+                _feedbackService.AddFeedback(feedback);
+                return Ok();
```

Comment:
200 OK is alright here, although 201 Created with the Location header set might be better for post? (I'm not asking to change this btw, just saying that your team should discuss which HTTP codes you'll be using and when exactly if you haven't yet.)